test/Serialization/attr-actorindependent.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -enable-experimental-concurrency -emit-module -o %t/a.swiftmodule -primary-file %s -module-name main
+// RUN: llvm-bcanalyzer -dump %t/a.swiftmodule | %FileCheck %s
+// RUN: %target-swift-frontend -enable-experimental-concurrency -emit-module -o %t/b.swiftmodule -primary-file %t/a.swiftmodule -module-name main
+// RUN: cmp -s %t/a.swiftmodule %t/b.swiftmodule
+
+///////////
+// This test checks for correct serialization & deserialization of
+// @actorIndependent and @actorIndependent(unsafe)
+
+// look for unsafe and safe versions of decl:
+
+// CHECK-NOT: UnknownCode
+// CHECK: <ActorIndependent_DECL_ATTR abbrevid=190 op0=1/>
+// CHECK: <ActorIndependent_DECL_ATTR abbrevid=190 op0=0/>
```

Comment:
We might that the abbrevid value will be brittle. You could replace the `190` with `{{[0-9]+}}`, because we don't really care what the value is.